.github/workflows/create-release-pr.yml

```
@@ -9,8 +9,8 @@ on:
       semver-version:
         description: 'A semantic version, eg: x.x.x'
         required: true
-      previous-semver-version:
-        description: 'Previous semantic version, eg: x.x.x'
+      previous-version-tag:
+        description: 'Previous release version tag. eg: v7.7.0'
```

Maybe we could call it a "ref" to make it more clear that it can be a tag or a hash?
```suggestion
      previous-version-ref:
        description: 'Previous release version reference (tag or commit hash). eg: v7.7.0'
```